app/src/main/java/com/codesoom/assignment/MyHandler.java

```diff
@@ -0,0 +1,144 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class MyHandler implements HttpHandler {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private final Map<Long, Task> taskMap = new LinkedHashMap<>();
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        URI uri = exchange.getRequestURI();
```

Comment:
가능한 한 재할당을 방지해 주도록 합시다.
```suggestion
        final String method = exchange.getRequestMethod();
        final URI uri = exchange.getRequestURI();
```

Reply:
궁금한 점이 있습니다.
메소드의 파라미터도 메소드 내에서 재할당 되지 않는다면 final keyword를 붙히는 것이 바람직한가요?
코드의 안정성과 컴파일러 최적화 관점에서 final keyword를 붙혀 재할당을 방지하는 것은 바람직한 코딩이라고 생각합니다만 재할당 되지 않는 모든 파라미터에 final을 붙히게 되면 코드가 장황해 보이지 않을까하는 생각도 듭니다.
실무에서는 어떤 방식으로 코딩을 하는지 궁금합니다.

Reply:
실무에서는 - 이건 사람마다 다 달라서 뭐라고 할 수 없어요. 어떤 사람은 안 붙이고 어떤 사람은 붙입니다. 그래서 정답이 없죠. 이건 자신만의 주관을 갖는 문제이기도 해요. 저는 가능한 한 final을 붙이는 방식으로 작업합니다. 하지만 이렇게 생각하지 않던 시절도 있었어요. @steve7867 님도 아마 계속 변하거나 아니면 하나의 스타일에 정착하시게 될 겁니다.